internal/pkg/service/buffer/store/export.go

```diff
@@ -270,6 +270,21 @@ func (s *Store) DeleteExport(ctx context.Context, exportKey key.ExportKey) (err
 		s.deleteExportBaseOp(ctx, exportKey),
 		s.deleteExportMappingsOp(ctx, exportKey),
 		s.deleteExportTokenOp(ctx, exportKey),
+		s.schema.ReceivedStats().InExport(exportKey).DeleteAll(),
+		s.schema.Files().Opened().InExport(exportKey).DeleteAll(),
+		s.schema.Files().Closed().InExport(exportKey).DeleteAll(),
+		s.schema.Files().Closing().InExport(exportKey).DeleteAll(),
+		s.schema.Files().Importing().InExport(exportKey).DeleteAll(),
+		s.schema.Files().Imported().InExport(exportKey).DeleteAll(),
+		s.schema.Files().Failed().InExport(exportKey).DeleteAll(),
+		s.schema.Slices().Opened().InExport(exportKey).DeleteAll(),
+		s.schema.Slices().Closing().InExport(exportKey).DeleteAll(),
+		s.schema.Slices().Uploading().InExport(exportKey).DeleteAll(),
+		s.schema.Slices().Uploaded().InExport(exportKey).DeleteAll(),
+		s.schema.Slices().Failed().InExport(exportKey).DeleteAll(),
+		s.schema.Records().InExport(exportKey).DeleteAll(),
+		s.schema.Tasks().InExport(exportKey).DeleteAll(),
+		s.schema.Runtime().LastRecordID().ByKey(exportKey).Delete(),
```

Comment:
Chybalo mazanie tychto prefixov, pri mazani exportu.
Casom to spravim mozno nejak krasie, aby to nebolo duplicitne aj v export aj v receiver.